Earthfile

```diff
@@ -267,6 +267,15 @@ base-image:
           systemctl enable cos-setup-network.service
     END
 
+    # TEST KAIROS-AGENT FROM BRANCH
+    ARG KAIROS_AGENT_DEV
+    ARG KAIROS_AGENT_DEV_BRANCH=master
```

Comment:
Not a strong opinion but can we use main ... most other repos I think are using main

Reply:
actually I just double checked and the agent branch is main so I think it will need to be the default